scripts/file_license_check.sh

```diff
@@ -2,7 +2,7 @@
 # run from the repo root
 
 rg -i "Copyright.*The Tari Project" --files-without-match \
-    -g '!*.{Dockerfile,asc,bat,config,config.js,css,csv,drawio,gitkeep,hbs,html,iss,json,lock,md,min.js,ps1,py,rc,scss,sh,sql,svg,toml,txt,yml,vue}' . \
+    -g '!*.{Dockerfile,asc,bat,config,config.js,css,csv,drawio,env,gitkeep,hbs,html,ini,iss,json,lock,md,min.js,ps1,py,rc,scss,sh,sql,svg,toml,txt,yml,vue,ts,tsx}' . \
```

Comment:
The `ts` and `tsx` are incorrect here, but that can be fixed in a different PR

Reply:
Sure.
It would also be handy to have a script inject the license metadata into files that don't have it